site/content/blog/markdowndb-launch.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Announcing the official launch of MarkdownDB! 🚀
```

Comment:
Suggest:

Announcing MarkdownDB: {Insert summary of what it is}

Why? because title is strongest thing we have - thing headline in a newspaper.

E.g.

Announcing MarkdownDB: an open source tool to create an SQL API to your markdown files

Reply:
Agree!